pipelines/pipeline.yml

```diff
@@ -13,89 +13,89 @@ pr:
   - release/*
 
 stages:
-  - stage: TestAndBuild
```

Comment:
commented out a bunch of the pipeline, just to make iteration faster while I'm working on this.